File: CodeForces/1701-1800/1760/A/1760A.swift
// Problem 1760A: Medium Number
// https://codeforces.com/contest/1760/problem/A
// Executed with: Swift version 6.2 (swift-6.2-RELEASE)
// Linux version: Linux kernel 6.17.0-5-generic
// Written on: October 26th, 2025
// Codeforces language used: Cannot be submitted

import Foundation

var runs = Int(readLine()!)!

for _ in 0...runs-1 {
    let nums = readLine()!.split(separator: " ").map { Int($0)! }
    let a = nums[0]
    let b = nums[1]
    let c = nums[2]
    
    if (a < b){
        if (a < c){
            print(min(b, c))
        } else {
            print(a)
        }
    } else {
        if (a > c) {
            print(max(b, c))
        } else {
            print(a)
        }
    }

}

File: CodeForces/1701-1800/1760/A/1760A.java
// Problem 1760A: Medium Number
// https://codeforces.com/contest/1760/problem/A
// Compiled with: openjdk 25 2025-09-16
// Executed with: javac 25
// Linux version: Linux kernel 6.17.0-5-generic
// Submitted on: October 25th, 2025
// Codeforces language used: Java 21 64bit

import java.util.Scanner;
 
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int runs = sc.nextInt();
        for (int i = 0; i < runs; i++) {
            int a = sc.nextInt();
            int b = sc.nextInt();
            int c = sc.nextInt();

            if (a < b){
                // Possible cases:
                // 1-2-3, 1-3-2, 2-3-1
                if (a < c){
                    //Possible cases:
                    // 1-2-3, 1-3-2
                    System.out.println(Math.min(b, c));
                } else {
                    //Possible cases:
                    // 2-3-1
                    System.out.println(a);
                }
            } else {
                // Possible cases:
                // 2-1-3, 3-1-2, 3-2-1
                if (a > c) {
                    // Possible cases:
                    // 3-1-2, 3-2-1
                    System.out.println(Math.max(b,c));
                } else {
                    // Possible cases:
                    // 2-1-3
                    System.out.println(a);
                }
            }
        }
        sc.close();
    }
}
